src/components/MedicalServiceMasterPanel.js

```diff
@@ -34,6 +34,8 @@ class MedicalServiceMasterPanel extends FormPanel {
   }
   render() {
     const { classes, edited, readOnly, isServiceValid, isServiceValidating, serviceValidationError} = this.props;
+    console.log(edited?.category);
+    console.log(typeof "" === null);
```

Comment:
Delete console logs.

Reply:
Thanks for pointing this out, it's deleted now